src/Service/ZGWToXxllncService.php

```diff
@@ -281,6 +300,7 @@ public function mapZGWToXxllnc(string $casetypeId, ObjectEntity $zaakTypeObject,
      */
     private function getZaakTypeId()
     {
+        return '4a2babc6-0a6d-4ae2-8b27-500cc80423c5';
```

Comment:
You always return the same id here, is this correct?

Reply:
Removed, good find